src/client/client4.js

```diff
@@ -1589,6 +1589,23 @@ export default class Client4 {
         );
     };
 
+    doPostActionWithCookie = async (postId, actionId, actionCookie, selectedOption = '') => {
```

Comment:
Probably we can do the same here rewrite `doPostAction` to simply call `doPostActionWithCookie` with the cookie value set to null, or something like that.